central/group/datastore/internal/store/store_impl.go

```diff
@@ -148,13 +159,28 @@ func addInTransaction(tx *bolt.Tx, group *storage.Group) error {
 		return errox.InvariantViolation.CausedBy(err)
 	}
 
-	return buc.Put([]byte(id), bytes)
+	if err := buc.Put([]byte(id), bytes); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
```suggestion
	return buc.Put([]byte(id), bytes)
```
: ). Same for update and delete.